build/Dockerfile.libbpf.base

```diff
@@ -9,18 +9,15 @@ ARG ARCH=$(uname -m)
 RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \ 
     yum install -y cpuid
 
-# install less frequently updated pkg first
-RUN yum install -y kmod xz python3 && yum clean all -y
+# enable CentOS Stream 9 repository
+RUN yum install -y https://rpmfind.net/linux/centos-stream/9-stream/BaseOS/aarch64/os/Packages/centos-gpg-keys-9.0-23.el9.noarch.rpm && \
+    yum install -y https://rpmfind.net/linux/centos-stream/9-stream/BaseOS/aarch64/os/Packages/centos-stream-repos-9.0-23.el9.noarch.rpm && \
+    yum clean all -y
 
-# dependencies for libbpf
```

Comment:
Please keep the libbpf dependencies and libraries. This dockerfile is used (later) by the kepler image. The sequence is:
base image -> build image -> kepler image

Since the libraries and dependencies takes time to download and build, we decide to split the base and kepler images and reuse the base image so we can save time on the kepler image build.

The idea of getting the tools into the base image is along that reusability line: we want to have a set of tools that are available when Kepler starts up. The best place is to put these tools in the base image so we don't have to download and install them every time Kepler image is built. Base image is the home for these tools.

For bcc and libbpf history, we started off with bcc library but hit a number of kernel version depenency issue so we are migrating to libbpf. At this time, bcc is still supported but with limited scope, new features don't need to go into bcc images. All new development are in libbpf images.

The perf and bpftrace tools are treated independent of bcc.

Reply:
Our current base image `docker.io/nvidia/cuda:12.1.1-base-ubi9` already contains `libbpf`. 
here is the logs while installing libbpf  inside docker image container:

```
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered with an entitlement server. You can use subscription-manager to register.

CentOS Stream 9 - BaseOS                        3.7 MB/s | 7.9 MB     00:02
CentOS Stream 9 - AppStream                     9.9 MB/s |  18 MB     00:01
CentOS Stream 9 - Extras packages                15 kB/s |  15 kB     00:00
cuda                                            1.6 MB/s | 1.0 MB     00:00
Red Hat Universal Base Image 9 (RPMs) - BaseOS  1.3 MB/s | 513 kB     00:00
Red Hat Universal Base Image 9 (RPMs) - AppStre 4.3 MB/s | 1.8 MB     00:00
Red Hat Universal Base Image 9 (RPMs) - CodeRea 608 kB/s | 190 kB     00:00
Package libbpf-2:1.2.0-1.el9.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
```

Please confirm from your side. If there are any doubts, I'll maintain these dependencies and retain the installation of libbpf within the Dockerfile.